ui/admin/app/styles/app.scss

```diff
@@ -1056,3 +1056,36 @@
     padding: 0.25rem 0.75rem;
   }
 }
+
+// worker filter generator
+.show-filter-generator {
+  .hds-form-field--layout-flag {
+    margin-bottom: 0;
+  }
+}
+
+.filter-generator-selection {
+  .hds-form-group__control-field {
+    margin-bottom: 0;
+  }
+}
+
+.formatted-results {
+  p {
+    margin-bottom: 0.5rem;
+  }
+}
+
+.generated-results-container {
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
+  align-items: flex-start;
+}
+
+// storage buckets
+.storage-bucket-worker-filter {
+  .hds-form-field--layout-flag {
```

Comment:
Same with this style, looks like it's our own style being overridden?

Reply:
This one we needed because the Hds::Separator adds top and bottom to itself and the margin-bottom from the global style made the spacing too much.